examples/amm/tealscript_artifacts/ConstantProductAMM.arc4.json

```diff
@@ -102,11 +102,11 @@
         },
         {
           "name": "aAsset",
-          "type": "asset"
+          "type": "uint64"
         },
         {
           "name": "bAsset",
-          "type": "asset"
```

Comment:
TealCraft shows AssetPicker in the user interface if the type is "asset".
With this change is it possible to identify between normal uint vs asset?

Other frameworks like Puya, Pyteal, Beaker also does the same ?